app/assets/src/components/SampleUpload.jsx

```diff
@@ -5,6 +5,8 @@ import $ from "jquery";
 import Tipsy from "react-tipsy";
 import IconComponent from "./IconComponent";
 import ObjectHelper from "../helpers/ObjectHelper";
+import Dropzone from "react-dropzone";
+import { Icon, Menu } from "semantic-ui-react";
```

Comment:
Can we not use semantic directly? 
We have Icon elements at least. What is the menu for?

Reply:
'Menu' for the button switcher. will change to use our Icon and wrap the Menu.